flink-connector-kafka/src/test/java/org/apache/flink/streaming/connectors/kafka/table/KafkaChangelogTableITCase.java

```diff
@@ -65,7 +64,6 @@ public void testKafkaDebeziumChangelogSource() throws Exception {
         tableConf.set(
                 ExecutionConfigOptions.TABLE_EXEC_MINIBATCH_ALLOW_LATENCY, Duration.ofSeconds(1));
         tableConf.set(ExecutionConfigOptions.TABLE_EXEC_MINIBATCH_SIZE, 5000L);
-        tableConf.set(OptimizerConfigOptions.TABLE_OPTIMIZER_AGG_PHASE_STRATEGY, "TWO_PHASE");
```

Comment:
What was the intention behind this change?

Reply:
The option went from String Flink 1.19 to Enum in Flink 1.20, so you can't compile the same code in both versions.
Talked to Timo what the option does and he said that it shouldn't apply here as we don't have aggregates. And lo-and-behold, nothing changed when I removed the line. (The other options are probably meaningless as well but I don't want to dig into the table tests).